stock_ownership_availability_rules/model/quant.py

```diff
@@ -29,15 +39,11 @@ def create(self, vals):
 
         """
         if not vals.get('owner_id'):
-            Company = self.env['res.company']
             location = self.env['stock.location'].browse(vals['location_id'])
 
             vals['owner_id'] = (
                 location.partner_id.id or
```

Comment:
I would put this line inside the method, and give the method a more generic name like `_get_default_owner`.